yarn.lock

```diff
@@ -1845,11 +1845,6 @@ dot-prop@^4.1.0:
   dependencies:
     is-obj "^1.0.0"
 
-dryrun@1.0.2:
-  version "1.0.2"
-  resolved "https://registry.yarnpkg.com/dryrun/-/dryrun-1.0.2.tgz#bf59e1193efea0de5a140ee89f6a89af81cee305"
-  integrity sha1-v1nhGT7+oN5aFA7on2qJr4HO4wU=
```

Comment:
How is this related to the `contentType` changes?

Reply:
It's not.
Artifact of not running `yarn` after getting rid of that package a few PRs back.

Reply:
Oh, yeah, didn't see that you'd added that in there. Come to think about it, it did disappear from my general cleanup branch when I rebased yesterday...